modules/tensor_mechanics/src/actions/TensorMechanicsAction.C

```diff
@@ -308,16 +325,13 @@ TensorMechanicsAction::act()
       _problem->addMaterial(type, name() + "_strain", params);
   }
 
-  //
-  // Add Stress Divergence Kernels
-  //
```

Comment:
why is this being removed?

Reply:
Because it's no longer true -- I'm adding another kernel as well. If I changed that to say "Add Kernels", that would be stating the obvious given the line below.

Reply:
Hm, those lines were less about explicitly stating what is done, but more about creating a visual spacer between these distinct tasks in this very long program text....